request_test.go

```diff
@@ -365,3 +365,18 @@ func TestHandleHTTPHEADRequest(t *testing.T) {
 	interceptor.HandleHTTP(w, req, handler)
 	st.Expect(t, string(w.Body), "Hi")
 }
+
+func TestHandleHTTPOPTIONSRequest(t *testing.T) {
+	modifierFunc := func(m *RequestModifier) {
+		m.Header.Set("foo", "bar")
+	}
+	w := utils.NewWriterStub()
+	req := &http.Request{Method: "OPTIONS", Header: make(http.Header)}
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprint(w, "Hi")
+		st.Expect(t, r.Header.Get("foo"), "")
```

Comment:
Damn, I supposed I was forgetting something :+1: . 

In any case, this opens another discussion. Do you think it makes sense to ignore those request methods? HEAD method is not too much used, but useful in REST or RPC models to perform health checks or similar verification only requests. OPTIONS could be useful to modify CORS headers and perform further checks. Since now the devs can add custom filters to ignore, or they can use `mux` package to compose middleware based on specific filters, so this could be useless and limiting for some devs. Opinions?